grc/gui/canvas/param.py

```diff
@@ -67,7 +67,10 @@ def format_tooltip_text(self):
         errors = self.get_error_messages()
         tooltip_lines = ['Key: ' + self.key, 'Type: ' + self.dtype]
         if self.is_valid():
-            value = str(self.get_evaluated())
+            value = self.get_evaluated()
+            if hasattr(value,"__len__"):
+                tooltip_lines.append('Length: {}'.format(len(value)))
+            vaule = str(value)
```

Comment:
This appears to be a typo

Reply:
Yes, well that was embarrassing. XD This is my first pull request...

Will comply with your suggestions.